components/MainNav.jsx

```diff
@@ -1,11 +1,22 @@
 import React from 'react';
 import styled from 'styled-components';
+import { useSpring, animated } from 'react-spring'
 import {
     BrowserRouter as Router,
     Link
 } from 'react-router-dom';
 
-const MainNav = ({ setIsNav, setIsSpeechBoxActive }) => {
+const MainNav = ({ setIsNav, isNav, setIsSpeechBoxActive }) => {
```

Comment:
I'm surprised to see isNav here. We had a boolean saying "if isNav is true, render this component (the one we're in now)." isNav could never be false while this component is rendered, so there cannot be any use to isNav in this component